### DISCUSSION TITLE
Created a PR in grafana/github-traffic, but not sure, whether it is being maintained

### DISCUSSION THREAD

**Discussion by GollyTicker** [grafana/grafana#53352] [Data Sources]
# RESOLVED

Hi everyone,

I am using [grafana/github-traffic](https://github.com/grafana/github-traffic) to collect and visualize the traffic for some of my public repositories.  I realised, that some metrics I am interested in were missing - so I added them and created this [pull request](https://github.com/grafana/github-traffic/pull/4).

I think, that the PR extension is reasonable and fits well into the repository. However, I have not got any response from the maintainer - which I think is https://github.com/dgzlopes - in the last two weeks. Looking at his recent github activity, I noticed, that he is setting back his maintainer status in [grafana/tempo](https://github.com/grafana/tempo/pull/1562) - and the same might be true for the github-traffic repository as well, since it's a relatively small and non-mature project.

My question hence is: Who can I contact for merging/reviewing this PR? Or am I too early in asking after 'just' two weeks?

This is my first discussion heere, so if I asked this question in the wrong place, please tell me where to ask instead, and I will do so.

Kinds!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
